front_end/html/header.html

```diff
@@ -24,6 +24,7 @@
       height: 80px;
           text-align: center;
     }
+
```

Comment:
I know this is nitpicky, but please remove this extra line. We don't have a blank line after the opening style tag, so let's remove this one to be consistent.

Reply:
It's fixed!